variables.tf

```diff
@@ -1,21 +1,7 @@
-variable "ddb_autoscale_read_schedule" {
```

Comment:
why did you remove the ddb scaling schedules?

Reply:
currently its not supported

Reply:
I do need to add a pr for that